crates/hdk/README.md

```diff
@@ -1,20 +1,9 @@
 # Holochain Development Kit (HDK)
 
-This HDK is currently in flux, expect rapid changes.
-
-There are a few big differences between this kit and previous kits.
```

Comment:
This feels like it's a necessary preamble to the following bullet points.

Reply:
I hear you, I think having the very first thing you read being a contrast with a previous way of doing things, is pretty unhelpful. Best to expand / reframe the thing as its intended to act and function now?

Reply:
@pdaoust what's your take on this one? do you want to propose something to resolve it, do you want me to try to write something instead? can we just skip it and merge?

Reply:
@Connoropolous Yeah, I hear what you're saying about leading with "this thing is better than that old thing you never used. Re-reading, I think it's just fine, actually. 👍